editor/src/generate_ts_types.rs

```diff
@@ -1,34 +1,15 @@
 /// Running this test will generate a `types.ts` file at the root of the repo,
 /// containing every type annotated with `specta::Type`
-// #[cfg(all(test, feature = "specta-export"))]
-#[ignore]
+
 #[test]
 fn generate_ts_types() {
```

Comment:
I fixed this but it's probally not needed with the wasm exporting solution (hence why it's still `#[ignore]`ed)